sound/soc/sof/nocodec.c

```diff
@@ -57,6 +60,8 @@ static int sof_nocodec_bes_setup(struct device *dev,
 			links[i].dpcm_playback = 1;
 		if (ops->drv[i].capture.channels_min)
 			links[i].dpcm_capture = 1;
+
+		links[i].be_hw_params_fixup = sof_pcm_dai_link_fixup;
```

Comment:
nit-pick on commit message: "override BE fixups" -? "set BE hw_params_fixup callback"?

I don't see any overwrite here.